packages/design-system/src/atoms/icon/index.tsx

```diff
@@ -46,6 +46,7 @@ import Layer from '../../assets/icons/layer.svg'
 import Link from '../../assets/icons/link.svg'
 import Link2 from '../../assets/icons/link-2.svg'
 import Lock from '../../assets/icons/lock.svg'
+import Logo from '../../assets/icons/roll-iso.svg'
```

Comment:
What do you think if we call it RollLogo? I believe that in the future we will have other brands' logos too (ex. Apple, Google, etc.)

```suggestion
import RollLogo from '../../assets/icons/roll-logo.svg'
```

Reply:
Yes, I agree that it should be more specific 👍🏻